wire-ios/Wire-iOS/Sources/Authentication/Interface/ViewControllers/AuthenticationHostingController.swift

```diff
@@ -82,6 +83,15 @@ final class AuthenticationHostingController<Content: View>: UIHostingController<
         authenticationCoordinator?.unauthenticatedSession.setErrorHandler {
             bridge.sendInboundEvent(.ssoAutheticationFailure)
         }
+
+        NotificationCenter.default
+            .publisher(for: AccountManagerDidUpdateAccountsNotificationName)
+            .sink { _ in
```

Comment:
suggestion: the notification's object should be the accountManager, so we can avoid accessing the singleton

Reply:
fixed